common-sns/src/main/java/org/duracloud/common/sns/config/SnsSubscriptionManagerConfig.java

```diff
@@ -56,9 +60,11 @@
                                + Inet4Address.getLocalHost()
                                              .getHostName()
                                              .replace(".", "_");
-            SnsSubscriptionManager subscriptionManager =
-                new SnsSubscriptionManager(new AmazonSQSClient(),
-                                           new AmazonSNSClient(),
+            AmazonSQS sqsClient = AmazonSQSClientBuilder.defaultClient();
+			AmazonSNS snsClient = AmazonSNSClientBuilder.defaultClient();
+			SnsSubscriptionManager subscriptionManager =
+                new SnsSubscriptionManager(sqsClient,
```

Comment:
There doesn't appear to be a need to create client variables here; creating the clients inline with the constructor would work just as well, i.e.

```
SnsSubscriptionManager subscriptionManager =
    new SnsSubscriptionManager(AmazonSQSClientBuilder.defaultClient(),
                               AmazonSNSClientBuilder.defaultClient(),
                               props.getInstanceNotificationTopicArn(),
                               ...
```
Also, indention of the line `SnsSubscriptionManager subscriptionManager =` needs to be pulled out.